blocks/blocks.go

```diff
@@ -124,6 +129,43 @@ func (bh *BlockHandler) HandleBlockData(ctx context.Context, data []byte, fetche
 	return nil
 }
 
+func (bh *BlockHandler) saveMetrics(blk types.Block) {
```

Comment:
there is no reason to make a method, right?

Reply:
yes